### DISCUSSION TITLE
Get attention

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [collab-community/journey-book#36] [General]
We require to get this project attention from Coding YouTubers, coders, devs (preferably experienced)

Any ideas how to get the attention of this project to people @vikasganiga05 @Adarsh-jaiss @TechStudent11

**Comment by Adarsh-jaiss** [comment]
-- firstly we can use this small introduction feature, in which folks can share a little about themselves and their social media handles (something like Eddie hub) and slowly slowly they will start sharing their Acheivents as well in their own readme 😄

**Reply by KendallDoesCoding** [reply]
good idea, but Eddie Hub only got that reach coz Eddie is a huge youtuber.

**Reply by Adarsh-jaiss** [reply]
-- yeah, but I can also do that but on small scale.. We have to make it as a pride for developers where they showcase there achievements... Are you getting what I am saying.

**Reply by KendallDoesCoding** [reply]
Yeah, exactly.

**Comment by Adarsh-jaiss** [comment]
--for small. Introduction, I can bring about Altlest 30- 40 folks who can give a small introduction about them

**Reply by KendallDoesCoding** [reply]
That'd be great.

**Reply by Adarsh-jaiss** [reply]
Yeah, but before that we need to finish this website thing... 
And no need to. Worry about users.. It's an open source projects, many folks are there who wanted to start but don't know how and what project they should first contribute to... We can recommend those about getting started their open source journey by contributing to this project.. 
And on Twitter learning in public is on hype, so we can use that as well because every contributor is going to share about his first achievement on Twitter and when other folks will see, they will also come and contri contribute and in this way, this project will gradually became one of the best project out there for folks

**Reply by KendallDoesCoding** [reply]
True!!

**Reply by Adarsh-jaiss** [reply]
So, don't take stress for user base, bringing user is my job 😄👍 
You guies just take care of backend 
I know marketing 😂😂

**Reply by KendallDoesCoding** [reply]
Gotcha!

**Reply by vikasganiga05** [reply]
Also, we can submit the project to review by Eddie on the live stream.

**Comment by KendallDoesCoding** [comment]
@Adarsh-jaiss , As the website is nearly done, request you to do the following.

- Create your own journey
- Tell friends and all coders to make a journey of their own in this repo, if we can get atleast 10-15 journeys within the next 3 or 4 days, that would be great.

@vikasganiga05 you too please make a journey of your own.

**Reply by Adarsh-jaiss** [reply]
Okay, but should we do that before the repository transfer or after the repository transfer 

Because it will lot more easy for folks in the community to contribute to community's GitHub rather than my GitHub profile 

So suggest me please @KendallDoesCoding 

Talking about the no. Of journeys - we can achieve that 15-20 journeys in just one day after the transfer of repo :)

**Comment by TechStudent10** [comment]
Looks like activity is back.

**Reply by Adarsh-jaiss** [reply]
Yeah

**Comment by KendallDoesCoding** [comment]
I was thinking now so we can get all the stuff on the website, later when
we transfer this repo there, the links will be redirected by GitHub, I
believe.

**Reply by Adarsh-jaiss** [reply]
Okay so, I will transfer the repo today itself with your permission gyus

**Reply by KendallDoesCoding** [reply]
Sure! Approved by me.

**Reply by vikasganiga05** [reply]
Approved by me too :+1:

**Comment by KendallDoesCoding** [comment]
@Adarsh-jaiss Please get the attention, put a announcement in the collab server!

**Reply by Adarsh-jaiss** [reply]
Already announced, but waiting for the folks
I'll be posting it for everyone very soon via Twitter :)

**Comment by KendallDoesCoding** [comment]
cool!
